README.md

```diff
@@ -169,8 +169,14 @@ this functionality.
 This job should be periodically run before end of a day. The following command creates `CronJob`
 scheduled 5 minutes before the midnight.
 
+For Cassandra, use:
 ```bash
-kubectl run jaeger-spark-dependencies --schedule="55 23 * * *" --env="STORAGE=cassandra" --env="CASSANDRA_CONTACT_POINTS=cassandra:9042"  --restart=Never --image=jaegertracing/spark-dependencies
+kubectl run jaeger-spark-dependencies --schedule="55 23 * * *" --env="STORAGE=cassandra" --env="CASSANDRA_CONTACT_POINTS=cassandra:9042" --restart=Never --image=jaegertracing/spark-dependencies
+```
+
+For Elasticsearch, use:
+```bash
+kubectl run jaeger-spark-dependencies --schedule="55 23 * * *" --env="STORAGE=elasticsearch" --env="CASSANDRA_CONTACT_POINTS=elasticsearch:9200" --env="ES_USERNAME=changeme" --env="ES_PASSWORD=changeme" --restart=Never --image=jaegertracing/spark-dependencies
```

Comment:
`env="CASSANDRA_CONTACT_POINT` it does not seem correct.

Reply:
ah, sorry. updated.